packages/server/public/css/items/note.css

```diff
@@ -1,3 +1,13 @@
+dt {
+	font-weight: bold;
+	margin-bottom: 0.25em;
+}
+
+dd {
+	margin-inline-start: 2.5em;
+	margin-bottom: 0.5em;
+}
+
```

Comment:
Can we do better than duplicating this code? And I thought that the published already used noteStyle.ts?

Reply:
You are right, the styles from noteStyles.ts are applied to to the shared notes. I'm removing the duplication.